odoo/tests/common.py

```diff
@@ -1153,6 +1153,16 @@ def navigate_to(self, url, wait_stop=False):
             self._websocket_wait_event('Page.frameStoppedLoading', params={'frameId': frame_id})
 
     def clear(self):
+        clear_service_workers = """
+        if ('serviceWorker' in navigator) {
+            navigator.serviceWorker.getRegistrations().then(
+                (registrations) => {registrations.map(r => r.unregister())}
```

Comment:
Why use a `map()` as you just need to iterate over registrations, not transform them?